tests/everest/functional/test_main_everest_entry.py

```diff
@@ -152,3 +157,37 @@ def test_everest_main_configdump_entry(copy_egg_test_data_to_tmp):
     assert render_dict["definitions"]["data_file"] == os.path.join(
         os.getcwd(), data_file
     )
+
+
+@pytest.mark.flaky(reruns=5)
+@pytest.mark.timeout(60)
+@pytest.mark.integration_test
+def test_stopping_local_queue_with_ctrl_c(copy_math_func_test_data_to_tmp):
+    config = EverestConfig.load_file(CONFIG_FILE_ADVANCED)
+
+    def wait_and_kill():
+        while True:
+            status = everserver_status(
+                ServerConfig.get_everserver_status_path(config.output_dir)
+            )
+            if status.get("status") == ServerStatus.running:
+                os.kill(os.getpid(), signal.SIGINT)
+                return
+            time.sleep(1)
+
+    thread = threading.Thread(target=wait_and_kill, args=())
+    thread.start()
+
+    with capture_streams() as (out, _), pytest.raises(SystemExit):
+        start_everest(["everest", "run", CONFIG_FILE_ADVANCED])
+
+    assert "You are running your optimization locally." in out.getvalue()
+    assert "KeyboardInterrupt" in out.getvalue()
+    assert (
+        "The optimization will be stopped and the program will exit..."
+        in out.getvalue()
+    )
```

Comment:
Suggest using the pytest fixture instead:
```suggestion
    with pytest.raises(SystemExit):
        start_everest(["everest", "run", CONFIG_FILE_ADVANCED])
    out = capsys.readouterr().out
    assert "You are running your optimization locally." in out
    assert "KeyboardInterrupt" in out
    assert (
        "The optimization will be stopped and the program will exit..."
        in out
    )
```